File: CS/Scaffolding.CodeFirst/Data/DepartmentContext.cs
using System.Data.Entity;

namespace Scaffolding.CodeFirst.Model {
    public class DepartmentContext : DbContext {
        public DbSet<Department> Departments { get; set; }
        public DbSet<Course> Courses { get; set; }
        public DbSet<Employee> Employees { get; set; }
    }
}

File: VB/Scaffolding.CodeFirst/Data/DepartmentContext.vb
Imports System.Data.Entity

Namespace Scaffolding.CodeFirst.Model
	Public Class DepartmentContext
		Inherits DbContext

		Public Property Departments() As DbSet(Of Department)
		Public Property Courses() As DbSet(Of Course)
		Public Property Employees() As DbSet(Of Employee)
	End Class
End Namespace